### ISSUE TITLE
OSC Controls

### ISSUE THREAD

**Issue by EdithRolle** [MiloMindbender/UE4VirtualProduction#13]
Hello,

Is it possible to use the OSC Controls, your blueprint, without using Vive controllers? I love the concept and would like to use it with my CG character during a broadcast.

thank you

**Comment by MiloMindbender** [comment]
Yes you can definitely do this. OSC is completely separate from any Vive stuff you can use the same blueprints I use to receive and respond to OSC commands in your own unreal actors
